src/vs/workbench/contrib/notebook/browser/viewParts/notebookKernelQuickPickStrategy.ts

```diff
@@ -72,7 +72,8 @@ export type KernelQuickPickContext =
 	{ id: string; extension: string } |
 	{ notebookEditorId: string } |
 	{ id: string; extension: string; notebookEditorId: string } |
-	{ ui?: boolean; notebookEditor?: NotebookEditorWidget };
+	{ ui?: boolean; notebookEditor?: NotebookEditorWidget } |
```

Comment:
every other user of this command is able to access the internal Editor widget or ID, but this usage would come from an extension